e2e/test_pipelines.py

```diff
@@ -0,0 +1,276 @@
+from os import access
```

Comment:
This file is incomplete and does not belong in this PR

Reply:
can u please remove it then